### DISCUSSION TITLE
OffScreen - Not loading full page

### DISCUSSION THREAD

**Discussion by AntonioSCoelho** [cefsharp/CefSharp#4407] [General]
Hi,

When running on Windows Server 2012 R2 the HTML page does not fully load, only partial. JavaScript's events on console just stops.
This issue started to happen only at the beginning of this month without any change on the server or application. This application has been running for months without any issue.
Can be reproduced on the `CEF` Sample Application and using the [MinimalExample](https://github.com/cefsharp/CefSharp.MinimalExample) as well. Testing the same HTML file on other Windows Versions run fine. Using Edge or Chrome to open the HTML run fine.

Has something been blocked on Windows Server 2012 R2 for `CEF`? Any Workaround? 

### Bug Report

- **What version of the product are you using?**
    Tested on v94.4.110 and 109.1.110

- **What architecture x86 or x64?**
    x64
	
- **What version of .Net?**
    .Net Framework 4.8
    
- **On what operating system?**
    Only Windows Server 2012 R2. Runs ok on any other earlier Windows Server version.

- **Are you using `WinForms`, `WPF` or `OffScreen`?**
    `OffScreen`
 
- **What is the expected output? What do you see instead?**
    Load full HTML page.

- **Please provide any additional information below.**
     Due to copyrights, can not provide sample code.

- **Please provide any additional information below.**
    No errors thrown on `CEF`.
 
- **Does this problem also occur in the `CEF` Sample Application**
    Same behaviour on `CEF` Sample Application. Using OffScreen parameters and without them.

**Comment by amaitland** [comment]
In future please use [Discussions](https://github.com/cefsharp/CefSharp/discussions) to ask support related  questions.

> This issue started to happen only at the beginning of this month without any change on the server or application. This application has been running for months without any issue.

Something must have changed. The `CefSharp` version you are using is fixed, contains everything to run.

Have you done a detailed analysis of your server? Windows updates? Firewall rule changes? Group policy changes? Antivirus updates?

> Using Edge or Chrome to open the HTML run fine.

Did you test  `Chrome` on the **actual** 2012R2 server?

> Has something been blocked on Windows Server 2012 R2 for `CEF`? Any Workaround?

Yours is the first report of issue I've seen. To me it sounds like something has changed in your environment. 

> * **Please provide any additional information below.**
>   No errors thrown on `CEF`.

Did you check the log file?

> * Same behaviour on `CEF` Sample Application.

As the problem reproduces with the `CEF` sample application you can try searching https://www.magpcss.org/ceforum/index.php
I don't recall seeing any issues that match this

There's a newer version you can try with https://cef-builds.spotifycdn.com/cef_binary_109.1.18%2Bgf1c41e4%2Bchromium-109.0.5414.120_windows64_client.tar.bz2
